### PR TITLE
fix: logo height

### PR DESCRIPTION
**Pull request by ghost**
Logo height change to 1.75
language selector dark:bg 
https://volta.net/nuxtlabs/nuxtjs.org-board/issues/76

### CODE REVIEW

File: components/atoms/HeaderLogo.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="flex items-center" @click.right.prevent="$router.push('/design')">
     <span>
-      <img :src="logo.light" class="w-auto h-6 md:h-8 dark:hidden" :alt="settings.title" />
-      <img :src="logo.dark" class="w-auto h-6 md:h-8 light:hidden" :alt="settings.title" />
+      <img :src="logo.light" class="w-auto h-6 md:h-7 dark:hidden" :alt="settings.title" />
+      <img :src="logo.dark" class="w-auto h-6 md:h-7 light:hidden" :alt="settings.title" />
```
Comment (Flosciante) [inline components/atoms/HeaderLogo.vue:5]: 👌🏻

File: components/molecules/footer/FooterLocaleSelector.vue [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <NuxtSelectNative
     v-model="locale"
     :options="locales"
-    select-class="appearance-none block w-full bg-none dark:bg-transparent light:bg-white border light:border-gray-200 dark:border-secondary-dark rounded-md py-2 pl-3 pr-10 text-base light:text-gray-500 dark:text-secondary-surface focus:outline-none light:focus:ring-black dark:focus:ring-white light:focus:border-gray-400 dark:focus:border-secondary-light sm:text-sm"
+    select-class="appearance-none font-sans block w-full d-body-bg border light:border-gray-200 dark:border-secondary-dark rounded-md py-2 pl-3 pr-10 text-base light:text-gray-500 dark:text-secondary-surface focus:outline-none light:focus:ring-black dark:focus:ring-white light:focus:border-gray-400 dark:focus:border-secondary-light sm:text-sm"
```
Comment (Flosciante) [inline components/molecules/footer/FooterLocaleSelector.vue]: Can you revert changes ? I'm working on it on another PR ;-)

**Review by Flosciante — CHANGES_REQUESTED** [review]

File: components/molecules/footer/FooterLocaleSelector.vue (hunk 2 above)
Comment (ghost) [reply components/molecules/footer/FooterLocaleSelector.vue]: Ok :)

**Review by Flosciante — APPROVED** [review]
thx

**Pull request merged**